packages/mdc-rtl/README.md

```diff
@@ -28,76 +26,20 @@ npm install --save @material/rtl
 
 ## Usage
 
-Simply `@import "@material/rtl/mixins";` and start using the mixins. Each mixin is described below.
+### Sass Mixins
 
-### mdc-rtl
+`$base-property` is a base box-model property. e.g. margin / border / padding.
+`$pos` is a horizontal position property, either "left" or "right".
 
-```scss
-@mixin mdc-rtl($root-selector: null)
-```
-
-Creates a rule that will be applied when an MDC-Web component is within the context of an RTL layout.
-
-Usage Example:
-
-```scss
-.mdc-foo {
-  position: absolute;
-  left: 0;
 
-  @include mdc-rtl {
-    left: auto;
-    right: 0;
-  }
+| Mixin | Description |
+| ----------------------------------------------- | - |
+| `mdc-rtl($root-selector)` | Creates a rule that is applied when the root element is within an RTL context |
+| `mdc-rtl-reflexive-box($base-property, $default-direction, $value, $root-selector)` | Applies the value to the `#{$base-property}-#{$default-direction}` property  in a LTR context, and flips the direction in an RTL context. This mixin zeros out the original value in an RTL context. |
```

Comment:
find a typo in comments here in `mdc-rtl/mixin.scss` line 128:
```
* You can also pass a 4th optional $root-selector argument which will be forwarded to `mdc-rtl`,
* e.g. `@include mdc-rtl-reflexive-box-property(margin, left, 8px, ".mdc-component")`.
```
It should be `@include mdc-rtl-reflexive-box`, we don't have a mixin called `mdc-rtl-reflexive-box-property`